lib/js/components/buttons/IconButton/IconButton.vue

```diff
@@ -31,61 +31,68 @@
 			:elevation="elevation"
 			:color="isButtonColor ? color : null"
 		>
-			<wnl-icon
-				class="a-iconButton__icon"
-				:icon="icon"
-				:size="iconSize"
-				:class="{ '-minor': colorScheme === ICON_BUTTON_COLOR_SCHEMES.MINOR_ICON }"
-			/>
+			<wnl-icon class="a-iconButton__icon" :icon="icon" :size="iconSize" />
 		</wnl-button>
 	</div>
 </template>
 
 <style lang="scss" scoped>
-@import '../../../styles/settings/animations';
-@import '../../../styles/settings/buttons';
-@import '../../../styles/settings/colors';
-@import '../../../styles/settings/icons';
-@import '../../../styles/settings/media-queries';
-@import '../../../styles/settings/spacings';
-@import '../../../styles/settings/typography';
-
-@mixin iconButtonColor($color, $hover-color: '') {
-	@if $hover-color == '' {
-		$hover-color: mix($color, $color-firefly-black, (1 - $button-hover-alpha) * 100%);
-	}
+@import '../../../../styles/settings/animations';
+@import '../../../../styles/settings/buttons';
+@import '../../../../styles/settings/colors';
+@import '../../../../styles/settings/icons';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/typography';
 
+@mixin iconButtonColor($color, $icon, $border, $background, $hovered, $focused, $disabled) {
 	color: $color;
+	border-color: $border;
+	background-color: transparent;
 
 	&:hover,
 	&:active {
-		color: $hover-color;
+		//background-color: $hovered;
```

Comment:
why these are commented out? Don't we need them to get correct colors when user interacts with this component?